content/applications/inventory_and_mrp/manufacturing/management/sub_assemblies.rst

```diff
@@ -2,48 +2,58 @@
 Manage semi-finished products
 =============================
 
-In Odoo, you can use subassembly products to simplify a complex *Bill
+With Odoo MRP, you can use semi-finished products to simplify a complex *Bill
 of Materials* or to represent your manufacturing flow more accurately.
-A *subassembly product* is a manufactured product that is used as a
-component to make another one.
+A *semi-finished product* is a manufactured product that is used as a
+component in a Bill of Material.
 
-A *BoM* that employs *subassemblies* is referred to as a multi-level
-BoM. Those are accomplished by creating a *top-level BoM* and
-*subassembly ones*. This process requires a route that will ensure
-that every time a manufacturing order for the top-level product is
-created, another one will be for subassemblies.
+A *BoM* that employs *semi-finished products* is referred to as 
+a multi-level BoM, where we distinguish between the *top level Product*
+and the *sub-assemblies*. 
 
-Configure the Subassembly Product
-=================================
+Configure a Multi Level BoM
+============================
 
-To configure a *multi-level BoM*, you will need a top-level product
-but also its subassemblies. The first step is to create a product form
-for each of the subassemblies. Select the routes *Manufacture* and
-*Replenish on Order*. Now, hit save.
+To configure a *multi-level BoM*, you will need the top-level product
+and its sub-assemblies. Therefore, you must first create the sub-assembly
+products and their respective Bill of Materials (https://www.odoo.com/documentation/15.0/applications/inventory_and_mrp/manufacturing/management/bill_configuration.html)
```

Comment:
Use the `doc` directive for relative paths: https://www.odoo.com/documentation/master/contributing/documentation/rst_cheat_sheet.html#the-doc-directive